Mage.Sets/src/mage/cards/i/InvadeTheCity.java

```diff
@@ -49,7 +49,7 @@ class InvadeTheCityEffect extends OneShotEffect {
     InvadeTheCityEffect() {
```

Comment:
Can this be the common Amass effect with `CardsInControllerGraveyardCount(StaticFilters.FILTER_CARD_INSTANT_AND_SORCERY)`? Probably still needs text override. (There are probably other cards that could be simplified in a similar way in the future.)

Reply:
Oh I did miss that one when searching for amass X.
Most of the others are amassing inside a `OneShotEffect`, along with other code and I didn't want to break any card.

Reply:
Oh there was Barad-Dur too. Now that I understand the power of `ManacostVariableValue.REGULAR`